_faq/security/does-istio-support-authorization.md

```diff
@@ -5,4 +5,4 @@ type: markdown
 ---
 {% include home.html %}
 
-Not currently - but we are working on it. At the moment, we only support the kubernetes service account as the principal identity in Istio Auth. We are investigating using [JWT](https://jwt.io/) together with mutual TLS to support enhanced authentication and authorization.
+Yes. Starting from Istio 0.5 release, we provide Role Based Access Control for services in Istio mesh. [Learn more](https://istio.io/docs/concepts/security/rbac.html).
```

Comment:
Please replace https://istio.io with {{home}}

Thanks.

Reply:
I wonder if we can check that from `rake test` somehow

Reply:
done